DABI.js

```diff
@@ -0,0 +1,677 @@
+};
+FW._Url = function() {
+    this._filters = new Array();
+    this.evaluate = function(d, c) {
+        var b = [c];
+        b = this._applyFilters(b, d);
+        if (b.length == 0) {
+            return false
+        } else {
+            return b
+        }
+    }
+};
+FW._Url.prototype = new FW._StringMagic();
+FW.Xpath = function(a) {
+    return new FW._Xpath(a)
+};
+FW._Xpath = function(a) {
+    this._xpath = a;
+    this._filters = new Array();
+    this.text = function() {
+        var b = function(c) {
+            if (typeof c === "object" && c.textContent) {
+                return c.textContent
+            } else {
+                return c
+            }
+        };
+        this.addFilter(b);
+        return this
+    };
+    this.sub = function(b) {
+        var c = function(f, e) {
+            var d = e.evaluate(b, f, null, XPathResult.ANY_TYPE, null);
+            if (d) {
+                return d.iterateNext()
+            } else {
+                return undefined
+            }
+        };
+        this.addFilter(c);
+        return this
+    };
+    this.evaluate = function(f) {
+        var e = f.evaluate(this._xpath, f, null, XPathResult.ANY_TYPE, null);
+        var d = e.resultType;
+        var c = new Array();
+        if (d == XPathResult.STRING_TYPE) {
+            c.push(e.stringValue)
+        } else {
+            if (d == XPathResult.ORDERED_NODE_ITERATOR_TYPE || d == XPathResult.UNORDERED_NODE_ITERATOR_TYPE) {
+                var b;
+                while ((b = e.iterateNext())) {
+                    c.push(b)
+                }
+            }
+        }
+        c = this._applyFilters(c, f);
+        if (c.length == 0) {
+            return false
+        } else {
+            return c
+        }
+    }
+};
+FW._Xpath.prototype = new FW._StringMagic();
+FW.detectWeb = function(e, b) {
+    for (var c in FW._scrapers) {
+        var d = FW._scrapers[c];
+        var f = d.evaluateThing(d.itemType, e, b);
+        var a = d.evaluateThing(d.detect, e, b);
+        if (a.length > 0 && a[0]) {
+            return f
+        }
+    }
+    return undefined
+};
+FW.getScraper = function(b, a) {
+    var c = FW.detectWeb(b, a);
+    return FW._scrapers.filter(function(d) {
+        return (d.evaluateThing(d.itemType, b, a) == c) && (d.evaluateThing(d.detect, b, a))
+    })[0]
+};
+FW.doWeb = function(c, a) {
+    var b = FW.getScraper(c, a);
+    b.makeItems(c, a, [], function(f, e, g, d) {
+        e.callHook("scraperDone", f, g, d);
+        if (!f.title) {
+            f.title = ""
+        }
+        f.complete()
+    }, function() {
+        Zotero.done()
+    });
+    Zotero.wait()
+};
+
+function detectWeb(doc, url) {
+    //namespace code
+    var namespace = doc.documentElement.namespaceURI;
+    var nsResolver = namespace ? function(prefix) {
+        if (prefix == 'x') return namespace;
+        else return null;
+    } : null;
+
+    if (doc.title.match("DABI. Datensatz Vollanzeige")) {
```

Comment:
`doc.title.indexOf("DABI. Datensatz Vollanzeige") == 0` or even `doc.title == "DABI. Datensatz Vollanzeige"` would be preferred. (avoid regexp when possible)